app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
```

Comment:
Standard library import and local application specific imports sould be separated by single blank line

Reply:
I agree with Ruslan